### ISSUE TITLE
Ensure CI validates RESP2 and RESP3

### ISSUE THREAD

**Issue by chayim** [redis/lettuce#2754]

**Comment by tishun** [comment]
Hey @sazzad16 , let me check that, but if we are talking about all commands then probably the answer is no.
Do we have that in other clients?

**Comment by sazzad16** [comment]
> Do we have that in other clients?

They either have that or are trying to have.

IMHO, at least all the data structure commands should be tested.

**Comment by tishun** [comment]
Makes sense, let's have this issue as a research and we can create a couple more once we find good targets to test
